### PR TITLE
Post-review wasm fixes

### PR DESCRIPTION
**Pull request by Manishearth**
followup from https://github.com/unicode-org/icu4x/pull/2757

### CODE REVIEW

**Review by sffc — COMMENTED** [review]
You should never run `npm i` in CI. That's an issue we need to fix

**Comment by Manishearth** [comment]
@sffc I don't know how to fix that, I don't understand npm enough. The problem is that `ava` is a binary that needs to be installed for our tests to run, and it gets installed with `npm i`.

File: .cargo/config.toml [hunk 1]
```diff
@@ -18,8 +18,10 @@ wasm-build-release = "panic-abort-build --target wasm32-unknown-unknown --profil
 
 [target.wasm32-unknown-unknown]
 rustflags = [
-  # 60 KiB, working around a bug in older rustc
+  # 100 KiB, working around a bug in older rustc
   # https://github.com/unicode-org/icu4x/issues/2753
   # keep in sync with wasm's build.sh
-  "-C", "link-args=-z stack-size=100000"
+  "-C", "link-args=-z stack-size=100000",
```
Comment (sffc) [inline .cargo/config.toml:24]: Observation: there is a space between the `z` and the `stack` here but not in build.sh

Comment (Manishearth) [reply .cargo/config.toml:24]: yeah, it doesn't matter, and it's easier to do it without the space in build.sh since it doesn't need to be quoted

**Pull request merged**